server/src/server.ts

```diff
@@ -74,6 +74,10 @@ connection.onInitialize(async (params: InitializeParams) => {
       connection.console.error('Cannot install prisma-fmt: ' + err)
     }
   }
+  connection.console.info(
+    'Installed version of Prisma plugin prisma-fmt: ' +
```

Comment:
I am not sure about the word "plugin"

Reply:
What do you think would be better here?

Reply:
Maybe just "Installed version of Prisma:". All binaries are and will be on the same version right? 

I haven't seen plugin been used with Prisma yet and in future it might be used in a different context.

Reply:
hmm I think saying only "installed version of Prisma" might be confusing with the actual version of Prisma, e.g. "0.0.40". How about "Installed version of binaries: "?

Reply:
"Prisma binary \`prisma-fmt`" maybe?

"plugin" is not the right term, and just "Prisma" is indeed confusing.